extensions/apiconnect/package.json

```diff
@@ -39,12 +39,15 @@
     "directory": "extensions/apiconnect"
   },
   "dependencies": {
-    "@loopback/core": "^2.9.1",
-    "@loopback/openapi-v3": "^3.4.5",
     "tslib": "^2.0.0"
   },
+  "peerDependencies": {
+    "@loopback/core": "^2.9.1",
+    "@loopback/rest": "^5.2.0"
+  },
```

Comment:
Should we wait for https://github.com/npm/rfcs/blob/latest/accepted/0025-install-peer-deps.md to be available in `npm@7`? 

At the moment, we have mixed usage of `peerDependencies` and `dependencies` for extension-like packages. It would be nice to make them consistent.

BTW, I also dislike the fact that we have to duplicate `peerDependencies` and `devDependencies`.

Reply:
> Should we wait for https://github.com/npm/rfcs/blob/latest/accepted/0025-install-peer-deps.md to be available in `npm@7`?

I think that's not needed. `lb4  app` always adds `@loopback/core` to project dependencies and `@loopback/rest` is usually added too (unless the user explicitly opts out).

BTW does it make any sense to use `@loopback/apiconnect` in a LB project (application) that does not have any controllers (no REST API)? Because if an app has REST API, then I think it must also have a dependency on `@loopback/rest`, right?

> At the moment, we have mixed usage of `peerDependencies` and `dependencies` for extension-like packages. It would be nice to make them consistent.

I agree! I thought we have a tracking issue for that, but could not find any. The closest one is https://github.com/strongloop/loopback-next/issues/5336. Should I open a new issue?
 
> BTW, I also dislike the fact that we have to duplicate entries in `peerDependencies` and `devDependencies`.

Yeah, I don't like that either. I guess it's the price we have to pay for using Node.js & npm.